src/games/PlayerVsPlayerGame.js

```diff
@@ -23,4 +31,44 @@ PlayerVsPlayerGame.prototype.switchMarker = function(playerMarker) {
   return playerMarker;
 }
 
+PlayerVsPlayerGame.prototype.endGame = function(gameBoard) {
+  return PlayerVsPlayerGame.prototype.winHorizontalTop(gameBoard) || PlayerVsPlayerGame.prototype.winHorizontalMiddle(gameBoard) ||
```

Comment:
I think this would read better as `return isWinner()` or something, and you move all of this into a private method that just returns the boolean as to whether or not there is a winner